force-app/main/default/classes/GiftBatch.cls

```diff
@@ -92,6 +92,17 @@ public inherited sharing class GiftBatch {
         set;
     }
 
+    @TestVisible
+    private PS_IntegrationServiceConfig paymentIntegrationServiceConfig {
```

Comment:
Not sure if the integration service belongs as a member of this class? Not quite sure what makes more sense here though